### PR TITLE
initial implementation

### PR DESCRIPTION
**Pull request by mickhansen**

### CODE REVIEW

File: src/Monitor.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+      }).filter(item => item);
+    }
+
+    let data = map(current, item => {
+      let tube = item.tube;
+
+      return {
+        tube: item.tube,
+        ...Object.keys(item).reduce((result, state) => {
+          if (~['tube', 'timestamp'].indexOf(state)) return result;
```
Comment (janmeier) [inline src/Monitor.js:116]: `.includes` ;)

Comment (mickhansen) [inline src/Monitor.js]: yep might as well!

Comment (mickhansen) [inline src/Monitor.js:116]: not supported by node 4, cba with babel for such a small thing :D

**Comment by janmeier** [comment]
Go go go :checkered_flag:

**Pull request merged**